portfolio/src/main/java/com/google/sps/servlets/CovidDataServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns covid data as a JSON object, e.g. {"2017": 52, "2018": 34}] */
+@WebServlet("/covid-data")
+public class CovidDataServlet extends HttpServlet {
+  
+  ArrayList<DateEntry> covidData = new ArrayList();
+  
+  private final class DateEntry {
+    private String date;
+    private LinkedHashMap<String, Integer> cases = new LinkedHashMap();
+
+    public DateEntry(String date) {
+      this.date = date;
+      this.cases = cases;
+    }
+    public void addEntry(String state, Integer cases) {
+      this.cases.put(state, cases);
+    }
+  }
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/us-states.csv"));
+    String currDate = "";
+    DateEntry temp = null; 
+
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
```

Comment:
Add a comment on what the shape of this looks like and what it represents.  cells[0] -> foo, cells[1] -> bar, etc.